src/django/api/views/facility/facilities_view_set.py

```diff
@@ -788,6 +789,12 @@ def delete_facility_list_item(item):
             item.facility = None
             item.save()
 
+        def delete_facility_claim_review_notes(claim):
```

Comment:
This function can be optimized.
```
def delete_facility_claim_review_notes(claim):
    FacilityClaimReviewNote.objects.filter(claim=claim).update(_change_reason=f'Deleted {claim.facility.id}')
    FacilityClaimReviewNote.objects.filter(claim=claim).delete()
```
By combining the update and delete operations into a single query, we can optimize the code and reduce the number of database queries executed.